### PR TITLE
document pages, improvements and bugfixes

### PR DESCRIPTION
**Pull request by peintnermax**

### CODE REVIEW

File: apps/login/src/lib/zitadel.ts [hunk 1]
```diff
@@ -52,17 +58,33 @@ export async function getBrandingSettings(organization?: string) {
...
+      return await settingsService
+        .getLoginSettings({ ctx: makeReqCtx(orgId) }, {})
+        .then((resp) =>
+          resp.settings
+            ? toJson(LoginSettingsSchema, resp.settings)
+            : undefined,
+        );
+    },
+    ["loginSettings", orgId ?? "default"],
+    { revalidate: 3600, tags: ["loginSettings"] },
```
Comment (eliobischof) [inline apps/login/src/lib/zitadel.ts:86]: Shouldn't we default to a very short cache or none at all in favor of DevX?

File: apps/login/src/ui/RegisterFormWithoutPassword.tsx [hunk 2]
```diff
@@ -63,10 +63,15 @@ export default function RegisterFormWithoutPassword({
       lastName: values.lastname,
       organization: organization,
     }).catch((error) => {
-      setError(error.message ?? "Could not register user");
+      console.error(error);
```
Comment (eliobischof) [inline apps/login/src/ui/RegisterFormWithoutPassword.tsx:66]:
```suggestion
```

File: apps/login/src/ui/SetPasswordForm.tsx [hunk 3]
```diff
@@ -66,32 +66,51 @@ export default function SetPasswordForm({
       authRequestId: authRequestId,
       password: values.password,
     }).catch((error: Error) => {
-      setError(error.message ?? "Could not register user");
+      console.error(error);
```
Comment (eliobischof) [inline apps/login/src/ui/SetPasswordForm.tsx:69]: Is this printed to the browser console? What's the point in not passing it to setError?

**Review by eliobischof — CHANGES_REQUESTED** [review]
Mostly typos

File: apps/login/src/lib/zitadel.ts (hunk 1 above)
Comment (peintnermax) [reply apps/login/src/lib/zitadel.ts:86]: I will introduce a global variable which can be overwritten as env variable 👍

File: apps/login/src/app/(login)/idp/page.tsx [hunk 4]
```diff
@@ -36,12 +30,12 @@ export default async function Page({
...
     <DynamicTheme branding={branding}>
       <div className="flex flex-col items-center space-y-4">
-        <h1>Register</h1>
+        <h1>Sign in with SSO</h1>
         <p className="ztdl-p">
-          Select one of the following providers to register
+          Select one of the following providers to sign in
         </p>
 
-        {legal && identityProviders && process.env.ZITADEL_API_URL && (
```
[the remark this answers is not in the document]
Comment (peintnermax) [reply apps/login/src/app/(login)/idp/page.tsx:44]: this was never needed in the IDP component. Yes the legal and help link will be shown somewhere but is not implemented yet.

File: apps/login/src/ui/SetPasswordForm.tsx (hunk 3 above)
Comment (eliobischof) [reply apps/login/src/ui/SetPasswordForm.tsx:69]: I mean the sensitive data is leaked to the dev console like this or am I missing something?`

Comment (peintnermax) [reply apps/login/src/ui/SetPasswordForm.tsx]: Yes we better remove it. I have revisited all console logs now besides the `navigator.credentials` requests which are not relevant

**Review by eliobischof — APPROVED** [review]

File: apps/login/src/ui/SetPasswordForm.tsx (hunk 3 above)
Comment (peintnermax) [reply apps/login/src/ui/SetPasswordForm.tsx]: server errors will not leak to a client component in nextjs btw. https://nextjs.org/docs/app/api-reference/file-conventions/error#errormessage

**Pull request merged**